### ISSUE TITLE
Dependency error with react 17

### ISSUE THREAD

**Issue by edelbalso** [AlexanderRichey/styled-react-modal#39]
Hi folks, our app is using react 17.0.1 and when I do an npm install, I see this error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: application-fe@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.9.0" from styled-react-modal@2.0.1
npm ERR! node_modules/styled-react-modal
npm ERR!   styled-react-modal@"^2.0.1" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/edu/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/edu/.npm/_logs/2021-04-03T19_02_17_773Z-debug.log
```

I'm not sure the impact of choosing `npm install --force` vs `npm install --legacy-peer-deps` for this, but curious what you folks would recommend as far as the right way of dealing with this.

**Comment by AlexanderRichey** [comment]
Hmm this usually results in a warning and not an error. I guess it's an error in this case because there are two different major versions of React. I think I can change `^16.9.0` to `^17.0.0 || ^16.9.0` in `package.json` to fix the issue. There's also a few housekeeping changes I want to make—I'm hoping to have some time this weekend to get this work done.

**Comment by AlexanderRichey** [comment]
This should now be fixed with the latest version 2.1.0.
